src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -104,6 +107,27 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
 
+    fire_and_forget Profiles::Icon_Click(IInspectable const&, RoutedEventArgs const&)
```

Comment:
we should file a follow-up to de-dupe these two. The only difference is which property ends up getting set